pyquil/api/_benchmark.py

```diff
@@ -37,17 +35,18 @@ class BenchmarkConnection(AbstractBenchmarker):
     """
 
     @_record_call
-    def __init__(self, *, client: Optional[Client] = None, timeout: float = 10):
+    def __init__(self, *, timeout: float = 5.0, client_configuration: Optional[QCSClientConfiguration] = None):
```

Comment:
Why the timeout change?

Reply:
I was aligning with the default for `httpx.Client`s, and 10 felt long from a UX perspective. But happy to hear differing thoughts.